From 149b54ef96d0aef6a19d1c65c2cef817b166fca4 Mon Sep 17 00:00:00 2001
From: 1aeo <tor@1aeo.com>
Date: Thu, 5 Mar 2026 05:07:10 +0000
Subject: [PATCH 3/7] Remove time.monotonic fallback and FIXME (available since
 Python 3.3)

The try/except fallback to time.time and the FIXME comment about using
ctypes are no longer needed. time.monotonic has been available since
Python 3.3, which is well below the minimum supported version.
---
 src/modules/rtt.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/modules/rtt.py b/src/modules/rtt.py
index 8ee79f6..cd08db3 100644
--- a/src/modules/rtt.py
+++ b/src/modules/rtt.py
@@ -43,11 +43,7 @@ import random
 import socket
 import util
 
-try:
-    from time import monotonic as tick
-except ImportError:
-    # FIXME: Maybe use ctypes to get at clock_gettime(CLOCK_MONOTONIC)?
-    from time import time as tick
+from time import monotonic as tick
 
 import selectors
 
-- 
2.43.0

